its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -442,4 +444,27 @@ void supportMavenEncryption() throws Exception {
     ORCHESTRATOR.executeBuild(build);
   }
 
+  @Test
+  void supportMavenEncryptionWithDefaultSecuritySettings() throws Exception {
+
+    wsClient.settings().set(new SetRequest().setKey("sonar.forceAuthentication").setValue("true"));
+    wsClient.users().create(new CreateRequest().setLogin("julien2").setName("Julien2").setPassword("123abc"));
```

Comment:
I am bit confused by the test here: is the password `123abc` or is `123abc` a reference to the encrypted password?

Reply:
the user's real password will be "123abc" Then we encrypted it and used the encrypted value in settings.xml. The decryptor should find you master password in settings-security.xml and decrypt it to be "123abc". That's what SecDispatchrer is actually doing.

The only difference between this test and the test above is that I don't specify the settings and expect the to be found in ~/.m2/setting-security.xml.

Looks like I should indeed need to refactor both tests to make them more clear.